keras_retinanet/bin/train.py

```diff
@@ -409,6 +409,11 @@ def csv_list(string):
     parser.add_argument('--config',           help='Path to a configuration parameters .ini file.')
     parser.add_argument('--weighted-average', help='Compute the mAP using the weighted average of precisions among classes.', action='store_true')
 
+    #Fit generator arguments
+    parser.add_argument('--multiprocessing', help='Fit generator mulitprocessing for n worker processes', action='store_true')
+    parser.add_argument('--workers', help='Number of multiprocessing workers', default=1)
```

Comment:
I'd like to see `--multiprocessing` and `--workers` combined into a single argument. A value of 0 (the default) could mean no multiprocessing, other values would indicate the number of workers.

Reply:
Looks good, thanks :)